### ISSUE TITLE
d3/svg support in jsPsych-html plugin

### ISSUE THREAD

**Issue by madebyafox** [jspsych/jsPsych#332]
using jsPsych 5.0.3 release package I am able to render svg via d3 in an external html page using the jsPsych-html plugin.   

Doesn't seem to work with the latest code base, however...  can't seem to trace down why

(scenario: running a learning science study examining graph comprehension -> rendering charts in d3 on external html pages and embedding them in the study using the jsPsych-html plugin.)

**Comment by jodeleeuw** [comment]
In the master branch `display_element` is no longer a jQuery object. It's a vanilla JavaScript HTMLElement, e.g.:

```javascript
display_element = document.querySelector('#jspsych-content')
```

**Comment by madebyafox** [comment]
Hmm... yeah, seems likely that would be related, though I'm not sure why that would change the behavior w/ d3. 

With master anything that gets added to the DOM via d3 in the external html page is not rendered, but with v5.0.3 it is.  This is case for any element type - not just adding an svg.  I also tried selecting and attaching the elements to a div constructed on the external html page rather than the body tag, but that doesn't work in master either.  

For now I'll proceed w/ v.5.0.3, but surely something to think about since d3 is so ubiquitous in data viz.

**Comment by jodeleeuw** [comment]
Yes, this is definitely something to consider. 

If you have a chance to post a small reproducible example it will help with testing going forward.

**Comment by jodeleeuw** [comment]
Hey @eweitnauer does your d3 code use jsPsych at all, or are you doing it externally? I am trying to figure out how to reproduce this issue.

**Comment by cogsmac** [comment]
I have a similar issue. I have worked through the tutorial for jsPsych, and have elements loading and data saving. 

I added another type of trial that simply calls an external-html page. It successfully displays text elements and a button but not the d3.js object itself. 

The example is in https://github.com/cogsmac/categoricalPerceptionVis under "psychShell-simpleGraphStim.html". The external page is "d3SampleBar.html". jsPsych is getting the page, but not displaying the d3.js bits. 

Hopefully this will serve as a reproducible example.
